pkgs/by-name/su/surrealist/package.nix

```diff
@@ -0,0 +1,66 @@
+{ autoPatchelfHook
+, cairo
+, dpkg
+, fetchurl
+, gdk-pixbuf
+, gobject-introspection
+, lib
+, libsoup
+, makeWrapper
+, pango
+, stdenv
+, webkitgtk
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
It doesn't seem like this uses the C toolchain, since we're downloading precompiled. Maybe this would work as well?

```suggestion
, stdenvNoCC
, webkitgtk
}:

stdenvNoCC.mkDerivation rec {
```

Reply:
Not necessarily. Sometimes the binary is autopatchelf'ed.